asllib/env.ml

```diff
@@ -36,12 +36,18 @@ module type S = sig
 
   module Scope : Backend.SCOPE
 
-  type global = { static : StaticEnv.global; storage : v Storage.t }
+  type global = {
+    static : StaticEnv.global;
+    storage : v Storage.t;
+    stack_size : int IMap.t;
```

Comment:
(* Binds each subprogram name to the number of calls to that subprogram that are currently under evaluation. *)